.github/workflows/copilot.yml

```diff
@@ -0,0 +1,59 @@
+name: Copilot Workflows
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+    inputs:
+        environment:
+          description:  Which AWS Account to use
+          type: choice
+          required: true
+          options:
+          - test
+        # Shared workflow consideration
+        # application:
+        #   description:  Application Name
+        #   type: string/choice
+        # - notification
+        #   required: true
+        init:
+          description: Initialise the application?
+          type: boolean
+          default: false
```

Comment:
What would this do? Is this some sort of meta workflow to create the copilot folder?